x-pack/legacy/plugins/apm/public/components/app/Main/route_config/index.tsx

```diff
@@ -134,7 +136,12 @@ export const routes: BreadcrumbRoute[] = [
     path: '/services/:serviceName/nodes/:serviceNodeName/metrics',
     component: () => <ServiceNodeMetrics />,
     breadcrumb: ({ location }) => {
-      const { serviceNodeName } = resolveUrlParams(location, {});
+      let { serviceNodeName } = resolveUrlParams(location, {});
+
+      if (serviceNodeName === SERVICE_NODE_NAME_MISSING) {
+        serviceNodeName = UNIDENTIFIED_SERVICE_NODES_LABEL;
```

Comment:
No need to re-assign:

```suggestion
        return UNIDENTIFIED_SERVICE_NODES_LABEL;
```